deepchecks/utils/correlation_methods.py

```diff
@@ -0,0 +1,111 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Module containing methods for calculating correlation between features."""
```

Comment:
change usages of "feature" to "variable" - feature is necessarily part of X, but these can be used for correlations with y as well.

Reply:
Good sugestion